eZ/Publish/Core/Base/Exceptions/ContentFieldValidationException.php

```diff
@@ -30,20 +32,35 @@ class ContentFieldValidationException extends APIContentFieldValidationException
      */
     protected $errors;
 
+    /** @var string|null */
+    protected $contentName;
+
     /**
      * Generates: Content fields did not validate.
      *
      * Also sets the given $fieldErrors to the internal property, retrievable by getFieldErrors()
      *
      * @param array<array-key, array<string, \eZ\Publish\Core\FieldType\ValidationError>> $errors
      */
-    public function __construct(array $errors)
+    public function __construct(array $errors, ?string $contentName = null)
     {
         $this->errors = $errors;
-        $this->setMessageTemplate('Content Fields did not validate');
+        $this->contentName = $contentName;
+
+        $this->setMessageTemplate('Content %contentName%fields did not validate: %errors%');
+        $this->setParameters([
+            '%errors%' => $this->generateValidationErrorsMessages(),
+            '%contentName%' => $this->contentName !== null ? sprintf('"%s" ', $this->contentName) : '',
+        ]);
+
         parent::__construct($this->getBaseTranslation());
     }
 
+    public static function createNewWithMultiline(array $errors, ?string $contentName = null): self
+    {
+        return new self($errors, $contentName);
```

Comment:
It doesn't provide any value as it is. You need to move whole logic from the constructor to `createNewWithMultiline` and leave the constructor as it was prior to this PR.